README.md

```diff
@@ -1,5 +1,7 @@
 # dotnetcore-console-sample
 
+![build status for all sample projects](https://github.com/briantjackett/dotnetcore-console-sample/workflows/build/badge.svg)
```

Comment:
Perhaps this should be a relative path?

Reply:
It probably should be relative.  Before I go there though, I also ran into an issue that the Microsoft Graph org on GitHub doesn't allow GitHub Actions.  As such can't run the build workflow that would supply the badge information for the upstream repo anyways.  I ended up doing a direct commit to the upstream master branch to remove this line from the README.  I left it in my personal fork for time being until I figure out a better way to support that without merging it back into upstream again.  Suggestions?

I was able to find multiple small issues in my personal fork (and upstream) by checking the build status of those checks which was great at the time.

Reply:
Spoke with @jthake-msft , there is active work to address GitHub Actions not available from this org.  Separate from that I'll look into making the badge image a relative link and will push new version once actions are available in this upstream repo.